migrations/versions/3fb61393a227_added_psi_enabled.py

```diff
@@ -0,0 +1,30 @@
+"""added psi enabled
+
+Revision ID: 3fb61393a227
+Revises: 290a91f4eff6
+Create Date: 2022-10-18 21:47:05.192322
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3fb61393a227'
+down_revision = '290a91f4eff6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('performance_profile', sa.Column('psi_enabled', sa.Boolean(), nullable=True))
```

Comment:
@saltgen  -
psi_enabled will be calculated based on availability of pressure metrics data. Will it make sense to keep default value as `False` and column as non-nullable?  
column value becomes True only when pressure metrics data available.

Reply:
@kgaikwad I took a look at all the non-nullable model fields and IMO `psi_enabled` doesn't qualify.
Also for some reason if we stop receiving value for the field or any respective record does not get updated
the value will be set to `False`, I think this has the potential to cause confusion in future.